src/list/manage/components/item-list-panel.js

```diff
@@ -38,6 +39,7 @@ export default function ItemListPanel({className, inputRef, totalItemCount,
         <ItemFilter className={styles.itemFilter}
                     inputRef={inputRef}/>
         <AddItem/>
+        <ListSort sort={sort} className={styles.listSort} {...props} />
```

Comment:
This layout puts everything into a single row.  Maybe we should consider some options:
* moving `<ListSort/>` (or a wrapper of it?) up a level to be a sibling of `<PanelHeader/>` and `<PanelBody/>`; or
* making `<PanelHeader/>`'s orientation configurable (flex column vs flex row)

I fear trying to make `<PanelHeader/>` a grid is more trouble than it's worth ...